### ISSUE TITLE
 Train SBS on Market1501 get very poor performance. 

### ISSUE THREAD

**Issue by yijliu7** [JDAI-CV/fast-reid#611]
I ran the command:
       python3 tools/train_net.py --config-file ./configs/Market1501/sbs_R50.yml --num-gpus 4

After 60 epochs, the loss dropped from 50 to roughly 30. 
mAP and Rank-N kept dropping during the whole training process. 

Any suggestions ?

**Comment by L1aoXingyu** [comment]
If you want to train with 4 GPUs, you need to change the batch_size x4.

**Comment by giaanthunder** [comment]
I'm facing the same issue. Did you solve it? Please share your solution

**Comment by nazmulnnb** [comment]
use PyTorch 1.8.1
